### PR TITLE
Example of Guard way protection for Scope

### PR DESCRIPTION
**Pull request by RoDmitry**
Example of Guard way protection for Scope

#### Description:

Docs update

#### Checklist:

- [ ] Tests for the changes have been added (_for bug fixes / features_);
- [ ] Docs have been added / updated (_for bug fixes / features_).
- [ ] This PR has been added to [CHANGELOG.md](https://github.com/DDtKey/actix-web-grants/blob/main/CHANGELOG.md) (to [Unreleased] section);

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,26 @@ App::new()
...
+            .to(|| async { HttpResponse::Ok().finish() })
+        ).guard(PermissionGuard::new("ROLE_ADMIN_ACCESS".to_string()))
+    ).service(
+        web::resource("/admin{regex:$|/.*?}").to(|| async { 
+            HttpResponse::TemporaryRedirect().append_header((header::LOCATION, "/login")).finish()
+        }))
+```
+When `Guard` lets you in the `Scope` (meaning you have `"ROLE_ADMIN_ACCESS"`), the redirect will be unreachable for you. Even if you will request `/admin/some_undefined_page`.
+
+Note: `regex` is a `Path` variable containing passed link.
```
Comment (DDtKey) [inline README.md:62]: Maybe it makes sense to attach a link to the `aсtix-web` documentation?

Comment (RoDmitry) [reply README.md:65]: You mean to guards? [This one](https://docs.rs/actix-web/3.3.2/actix_web/guard/index.html)?

Comment (DDtKey) [reply README.md:65]: I meant `Path` 🙂

Comment (RoDmitry) [reply README.md:65]: I just don't think that it's much usable in this case. But yeah we can add the [link](https://docs.rs/actix-web/3.3.2/actix_web/web/fn.resource.html)

**Comment by DDtKey** [comment]
Thank you very much! 👏 
I have merged PR & updated readme structure.

**Pull request merged**